static/js/index.js

```diff
@@ -589,6 +589,7 @@ EpComments.prototype.insertComment = function (commentId, comment, index) {
 
   comment.commentId = commentId;
   comment.reply = true;
+  comment.changeArgs = JSON.stringify({changeFrom: comment.changeFrom, changeTo: comment.changeTo}).replace(/"/g, '&quot;');
   content = $('#commentsTemplate').tmpl(comment);
```

Comment:
I'm not very familiar with this plugin, but I'm a bit worried about mutating the caller's `comment` object. What do you think about this instead?

```suggestion
  content = $('#commentsTemplate').tmpl({
    ...comment,
    changeArgs: JSON.stringify({
      changeFrom: comment.changeFrom,
      changeTo: comment.changeTo,
    }).replace(/"/g, '&quot;'),
  });
```

Or maybe it would be better to let the browser handle escaping:

```suggestion
  content = $('#commentsTemplate').tmpl(comment);
  content.find('.from-label')[0].dataset.l10nArgs = JSON.stringify({
    changeFrom: comment.changeFrom,
    changeTo: comment.changeTo,
  });
```